src/NotificationCenterSdk/Subscribers/SnoozeRequest.cs

```diff
@@ -0,0 +1,20 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace NotificationCenterSdk.Subscribers;
+
+public class SnoozeRequest
+{
+    public string SubscriberId { get; set; }
+    public SnoozeRequestBody Body { get; set; } 
+
+
+    public class SnoozeRequestBody
```

Comment:
rename to SnoozeNotifcationRequestBody if you found any other appropriate name instead of Body let me know